extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -131,7 +132,7 @@ export class InstrumentationService {
   addListener(config, listener) {
     const eventType = config['on'];
     if (eventType === AnalyticsEventType.VISIBLE) {
-      this.createVisibilityListener_(listener, config);
+      this.createVisibilityListener_(listener, config, /* visible */ true);
```

Comment:
updated the PR. I did not change visibility-impl to also accept the eventType because it didn't make a whole lot of sense. I did update the names and description to clarify what the code is trying to do.